test/integration/smoke/test_vm_autoscaling.py

```diff
@@ -0,0 +1,1103 @@
+        self.message("Waiting %s seconds for other %s VM(s) to be created" % (sleeptime, 1))
+        time.sleep(sleeptime)
+        self.verifyVmCountAndProfiles(MAX_MEMBER + 1)
+
+    @attr(tags=["advanced"], required_hardware="false")
+    def test_03_scale_down_verify(self):
+        """ Verify scale down of AutoScaling VM Group """
+        self.message("Running test_03_scale_down_verify")
+
+        self.autoscaling_vmgroup.disable(self.regular_user_apiclient)
+
+        policies = Autoscale.listAutoscalePolicies(
+            self.regular_user_apiclient,
+            action="ScaleUp",
+            vmgroupid=self.autoscaling_vmgroup.id
+        )
+        self.assertEqual(
+            isinstance(policies, list),
+            True,
+            "List autoscale policies should return a valid list"
+        )
+        self.assertEqual(
+            len(policies) == 1,
+            True,
+            "The number of autoscale policies (%s) should be equal to 1" % (len(policies))
+        )
+        scale_up_policy = policies[0]
+
+        conditions = Autoscale.listConditions(
+            self.regular_user_apiclient,
+            policyid=scale_up_policy.id
+        )
+        self.assertEqual(
+            isinstance(conditions, list),
+            True,
+            "List conditions should return a valid list"
+        )
+
+        for condition in conditions:
+            if condition.counterid == self.counter_cpu_id:
+                Autoscale.updateCondition(
+                    self.regular_user_apiclient,
+                    id=condition.id,
+                    relationaloperator="GT",
+                    threshold=101
+                )
+
+        policies = Autoscale.listAutoscalePolicies(
+            self.regular_user_apiclient,
+            action="ScaleDown",
+            vmgroupid=self.autoscaling_vmgroup.id
+        )
+        self.assertEqual(
+            isinstance(policies, list),
+            True,
+            "List autoscale policies should return a valid list"
+        )
+        self.assertEqual(
+            len(policies) == 1,
+            True,
+            "The number of autoscale policies (%s) should be equal to 1" % (len(policies))
+        )
+        scale_down_policy = policies[0]
+
+        new_condition_1 = AutoScaleCondition.create(
+            self.regular_user_apiclient,
+            counterid=self.counter_network_received_id,
+            relationaloperator="GE",
+            threshold=0
+        )
+        new_condition_2 = AutoScaleCondition.create(
+            self.regular_user_apiclient,
+            counterid=self.counter_network_transmit_id,
+            relationaloperator="GE",
+            threshold=0
+        )
+        new_condition_3 = AutoScaleCondition.create(
+            self.regular_user_apiclient,
+            counterid=self.counter_lb_connection_id,
+            relationaloperator="GE",
+            threshold=0
+        )
```

Comment:
not adding these to `self.cleanup`?

Reply:
moved to setUpClass and added to cls._cleanup